README.md

```diff
@@ -62,8 +62,34 @@ var Fetcher = require('fetchr'),
     })
 //...
 ```
+#### 3. API CORS Support
+You can pass the full origin host into `corsPath` and potentially overwrite `constructGetUri`. For example:
 
-### 3. Register data fetchers for API and/or DB access
+```js
+var qs = require('qs');
+function constructGetUri (uri, resource, params, config) {
+	// this refers to the Fetcher object itself that this function
+	// is invoked with.
+	if (config.cors) {
+		return uri + '/' + resource + '?' + qs.stringify(this.context);
+	}
+	
+	return this.constructGetUri.apply(this, arguments);
```

Comment:
I don't really like letting the user knowing about and calling an internal function. Could we just have the user return a `falsy` value and we will automatically call the internal default function for them as fallback?